index.js

```diff
@@ -98,7 +107,10 @@ function exec(str, cols, opts) {
 	const words = str.split(' ');
 	const rows = [''];
 
-	for (let i = 0, word; (word = words[i]) !== undefined; i++) {
+	for (const item of Array.from(words).entries()) {
+		const i = item[0];
+		const x = item[1];
```

Comment:
Can you use a more descriptive variable than `x` here? Since the scope is large, it makes it hard to read the code.